index.scss

```diff
@@ -58,12 +59,13 @@ body{
     }
   }
   @media screen and (max-width: 500px) {
-    .allUserCardContainer {
-      width: 90vw;
-      .userCard {
-        width: 90vw;
-      }
-    }
+    //.allUserCardContainer {
+    //  padding: 30px;
+    //  }
+      //.userCard {
+      //  width: 100%;
+      //  margin: 35px;
+      //}
   }
```

Comment:
everything in this media query is commented out - surely the media query should be deleted altogether?